### DISCUSSION TITLE
Disk images won't copy Winblowz <--> Linux

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#85] [General]
Either direction, device access error on eq005.

I finally found my old Linux ssd, updated it, and got dtcyber node install full to actually complete (had to give up on winblowz for that, relying on my previous install instead).

This one is throwing me, you would think an image is an image, and not tied to anything modern, such as a host os.

Both from same sources.![Xs6Ixa4S5P_052530.png](https://github.com/kej715/DtCyber/assets/136194499/07b4f7fe-02ff-41e7-893c-6c1f21536e19)

Same cyber.ini and site.cfg in both win and Linux.  Both do just fine on native built images, but do this identically when using the disks of the other.  Thought it couldn't handle symlinks to the NTFS tapes and disks dirs (2.8.7) yesterday, but no, the disk images of one do not work with the other.  Site.cfg is just name changes at this point.

Pictured above is an attempt to deadstart the freshly built images copied over from the Linux side.  Could this be a NTFS problem in Linux alternatively?  I have r-studio, so I can copy the file from ext4 into win and test that direction, but I don't think thst is it, because i tarballed/bzipped tapes/* and disks/* and under win it extracted error-free.

Well, clean build on the Linux side now...progress...

**Comment by wschaub** [comment]
EQ5 and EQ6 are UEM and ECS memory. not disk images. those live inside persistence/* and not disk/* 

If you did copy over disk images the contents should be recovered just fine by a level 0 deadstart. 

You can try either copying over persistence/* as well and see if it comes up correctly, or just re-initialize equipments 5 and 6 by
doing ../dtcyber manual
selecting option O, P, D=YES and then doing NEXT. INITIALIZE,AL,5,6. and then GO.

**Reply by KC5VDJ** [reply]
Thanks .  That saves me the alternative of copying the cached install zips/tapes over to the winblowz side to play with the install scripts for offline / online-but-nothing-modded bits...which I'll do anyway once I have this duplicated on the win side.

**Comment by KC5VDJ** [comment]
Yup, it was definitely my brainfarting on persistence...

Copied over fine, and now running in winblowz.

Thanks!

**Comment by KC5VDJ** [comment]
Now have tapes disks and persistence symlinked over to the win disk, and running nice under Linux now.   That'll free up a little space, yet allow a common system regardless of host os

**Comment by KC5VDJ** [comment]
The install full did what I thought, and I can now run cybis reliably, without the kludge.  I thought it would do the trick, installing from scratch on the 4MW config.

Had to restart a couple of times due to unreliable networking on my side (flaky wifi), but no actual errors.  Got a beginning to end print listing too (tail -n 100000 --retry -f  LP5xx_C12_E5 |tee install.out.txt) (or rm then touch the LP file, and leave out the -n 100000, THEN start node install full)
